app/services/clients/connector_client.js

```diff
@@ -55,7 +55,7 @@ const _getPatchUrlFor = chargeId => baseUrl + CARD_CHARGE_PATH.replace('{chargeI
 const _getWorldpay3dsFlexUrlFor = chargeId => baseUrl + WORLDPAY_3DS_FLEX_JWT_PATH.replace('{chargeId}', chargeId)
 
 /** @private */
-const _putConnector = (url, payload, description, loggingFields = {}) => {
+const _putConnector = (url, payload, description, loggingFields = {}, callingFunction) => {
```

Comment:
There are ~6 anonymous functions here that are complex and should be named. Happy for that to be done separately.

Reply:
There's some more clean up to do, I'll do this in the next PR.